src/sentry/runner/commands/run.py

```diff
@@ -504,3 +504,20 @@ def ingest_consumer(consumer_types, all_consumer_types, **options):
         ingest_consumer_types=",".join(sorted(consumer_types)), _all_threads=True
     ):
         get_ingest_consumer(consumer_types=consumer_types, executor=executor, **options).run()
+
+
+@run.command("metrics-consumer")
+@log_options()
+@click.option(
+    "--group_id",
+    default="metrics-consumer",
+    help="Consumer group to track metric indexer offsets. ",
+)
+@click.option("--topic", default="ingest-metrics", help="Topic to get subscription updates from.")
+@batching_kafka_options("metrics-consumer")
+@configuration
+def metrics_consumer(**options):
```

Comment:
Is this going to start the consumer by default? I don't think we need it yet by default.

Reply:
I've had to manually run `sentry run metrics-consumer` so I don't think this starts by default

Reply:
How come we are adding a separate command rather than using the single `ingest-consumer` command which runs the rest of the ingest consumers? We could still temporarily omit from `--all-consumer-types` and only run it if explicitly called with the metrics consumer type if was the concern. Curious if there is another reason.

Reply:
@lynnagara (cc @fpacifici) the `ingest-consumer` commands ends up using the `IngestConsumerWorker` where as we want to use the `MetricsIndexerWorker` instead. I felt like it was easier to keep these separate for now then to refactor the `ingest-consumer` command. It seems like this could be easily changed down the line if we wanted, but open to changing it now if people feel strongly